### DISCUSSION TITLE
Vil ikke beregne "Der skete en fejl i forsøget på at starte maxima Errcode: 2"

### DISCUSSION THREAD

**Discussion by Silujo** [Eduap-com/WordMat#65] [Support] [answered]
Efter at have opdateret wordmat kan jeg ikke længere udregne selv simple beregninger. 
Jeg får blot denne besked: "Der skete en fejl i forsøget på at starte maxima Errcode: 2". 

Jeg har læst mig til at det kan være mit antivirusprogram, der er i vejen. Dette har jeg derfor slettet. 
Derudover har jeg opdateret min PC til windows 11, opdateret microsoft, samt forsøgt at installerer den tidligere version af wordmat, som jeg ikke tidligere havde dette problem med. Men jeg kan bare ikke få det til at virke... 

Nogen forslag? 
Håber virkelig på en hjælpende hånd - Tak :-) 

![image](https://user-images.githubusercontent.com/105111370/167254370-b9143439-3c19-43b7-9805-862ca7884197.png)

/ Signe

**Comment by Eduap-com** [comment]
Har set denne fejl på enkelte computere, men altid i forbindelse med antivirus. Hvis det helt er fjernet ved jeg ikke hvad problemet er. 
Mit forslag er at skifte til GeoGebra som CAS-motor. Det plejer at virke.

**Comment by Eduap-com** [comment, marked as answer]
Se også denne tråd: [https://github.com/Eduap-com/WordMat/discussions/66](url)
